resources/networks.json

```diff
@@ -1,15 +1,15 @@
 {
-	"testnet": {
+	"mumbai": {
 		"envDescription": "Public testnet environment where users can interact with Nevermined protocol.",
 		"envUrl": "https://nevermined.io/",
 		"isProduction": false,
 		"nvm": {
 			"nodeUri": "",
```

Comment:
should we have a default `nodeUri` for all networks using public endpoints taken from https://chainlist.org/chain/80001